llvm/lib/Target/RISCV/RISCVInstrInfo.cpp

```diff
@@ -43,6 +44,10 @@ using namespace llvm;
 #define GET_INSTRINFO_NAMED_OPS
 #include "RISCVGenInstrInfo.inc"
 
+#define DEBUG_TYPE "riscv-instr-info"
+STATISTIC(TotalLMULSpilled, "Total LMUL spilled from vector registers");
```

Comment:
Rather than describing this as LMUL, please describe in terms of vector registers (i.e. not register groups).  Something like "Number of registers within vector register groups spilled" or something along those lines.